StepsToCare/CareDashboard.ascx.cs

```diff
@@ -886,7 +886,7 @@ public void gList_RowDataBound( object sender, System.Web.UI.WebControls.GridVie
                                 {
                                     e.Row.AddCssClass( "assigned" );
                                 }
-                                if ( assignedPerson.WorkerId.HasValue && assignedPerson.FollowUpWorker.HasValue && assignedPerson.FollowUpWorker.Value )
+                                if ( assignedPerson.WorkerId.HasValue && assignedPerson.FollowUpWorker )
                                 {
                                     assignedFollowUpWorker = assignedPerson;
```

Comment:
Does this need to take into account multiple follow up workers? It only assigns to one?